blobfuse/tests.py

```diff
@@ -1118,6 +1118,32 @@ def test_file_simultaneous_write(self):
         
         os.close(fd)
         os.remove(testFilePath)
+
+
+    def test_multiple_file_handles_on_single_file(self):
+        # This is to test whether close of a file handle impacts other open file handles on the same file
+        # reported in issue 57
+        testFileName = "testFile_new" + str(random.randint(0, 10000))
+        testFilePath = os.path.join(self.blobstage, testFileName)
+
+        repeat = random.randint(1, 10)
+        fd1 = os.open(testFilePath, os.O_WRONLY | os.O_CREAT)
+        fd2 = os.open(testFilePath, os.O_WRONLY)
+        os.close(fd2)
+       
+        # sleep until cache times out 
+        time.sleep(130)
```

Comment:
This will more than double the length of the test pass.  Please add a # TODO to fix this :)